app/controllers/api/v1/recipes_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::RecipesController < Api::V1::BaseController
   before_action :set_recipe, only: %i[show update create upload_img add_review]
   def index
-    #params[:query] = "eg"
+    params[:query] = "eg"
```

Comment:
this is for personal testing?
please remove

Reply:
sure!